test/OpenTelemetry.Tests/Metrics/MetricExemplarTests.cs

```diff
@@ -52,6 +53,14 @@ public class MetricExemplarTests : MetricTestsBase
 
         Assert.NotNull(meterProviderSdk);
         Assert.Equal((ExemplarFilterType?)expectedValue, meterProviderSdk.ExemplarFilter);
+        if (programmaticValue.HasValue)
```

Comment:
Could we add a test to validate the precedence between `OTEL_METRICS_EXEMPLAR_FILTER` and `OTEL_DOTNET_EXPERIMENTAL_METRICS_EXEMPLAR_FILTER_HISTOGRAMS`

Reply:
I tweaked this test to verify the histogram key takes precedence over the global one: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5611/commits/50f3eddff516d41397bb59dc3bf7b153dacc93f2